.github/workflows/build-binaries.yml

```diff
@@ -147,6 +149,7 @@ jobs:
           bodyFile: ${{ env.SHOULD_PUBLISH_ALPHA == 'true' && 'build/release-notes-alpha.md' || 'build/release-notes.md' }}
           artifacts: 'dist/latest-linux.yml'
           allowUpdates: true
+          omitBodyDuringUpdate: true
```

Comment:
I think I didn't put `omitBodyDuringUpdate` here because we needed one job to upload the release notes?

Reply:
removed